language-server/modules/langserver-core/src/test/resources/codeaction/create-function/source/create_function_in_explicit_anonymous_function.bal

```diff
@@ -0,0 +1,20 @@
+import ballerina/module1;
+
+function createFunctionInWorker() {
+    var fn1 = function() returns int {
+        return foo();
+    };
+
+    var fn2 = function() returns string {
+        foo(32);
```

Comment:
Shall we add another test by wrapping with statement inside another lambda function?